### PR TITLE
minor README.auxIO updates

### PR DESCRIPTION
**Pull request by mppf**
Minor formatting, cross-referencing cleanups.

### CODE REVIEW

File: doc/release/technotes/README.auxIO.rst [hunk 1]
```diff
@@ -12,8 +14,13 @@ systems simultaneously.
 Dependencies
 ------------
 
-For each AIO system described here, there is a corresponding README.<system> in
-this directory detailing how to install and use them.
+Each AIO system is typically described in the support module for it.
+See:
+
+ * :mod:`HDFS`
```
Comment (ben-albrecht) [inline doc/release/technotes/README.auxIO.rst]: What is `:mod:` doing and do you intend for these to be blue links? (They are in Github render)

Comment (mppf) [inline doc/release/technotes/README.auxIO.rst]: That makes a cross-reference to the Chapel module called HDFS. It works for me (in my rendered HTML).

Comment (ben-albrecht) [inline doc/release/technotes/README.auxIO.rst]: Okay. Github version is not as important.

**Comment by ben-albrecht** [comment]
Everything looks good. Nice edits. :+1:

File: doc/release/technotes/README.auxIO.rst (hunk 1 above)
Comment (mppf) [inline doc/release/technotes/README.auxIO.rst]: If it becomes a problem, feel free to refer to the HDFS module some other way. I do this :mod: thing in a few other READMEs too. I had to put .. default-domain:: chpl at the top of the file to get it to work.

**Pull request merged**